pcsd/locales/en-US.yml

```diff
@@ -0,0 +1,192 @@
+  notConnected: "Not Connected"
+  status: "Status"
+  start: "Start"
+  stop: "Stop"
+  restart: "Restart"
+  unstandby: "Unstandby"
+  unmaintenance: "Unmaintenance"
+  configureFencing: "Configure Fencing"
+  addExisting: "Add Existing"
+  destroy: "Destroy"
+  running: "Running"
+  stopped: "Stopped"
+  unknown: "Unknown"
+  notAuthorized: "not Authorized"
+  attribute: "Attribute"
+  levels: "Levels"
+  level: "Level"
+  addresses: "Addresses"
+  watchdog: "Watchdog"
+  watchdogs: "Watchdogs"
+  devices: "Devices"
+  device: "Device"
+  cluster: "Cluster"
+  errors: "Errors"
+  warnings: "Warnings"
+  unmanaged: "Unmanaged"
+  transport: "Transport"
+  links: "Links"
+  link: "Link"
+  and: "and"
+  options: "Options"
+  compression: "Compression"
+  crypto: "Crypto"
+  default: "Default"
+  port: "Port"
+  ports: "Ports"
+  configuration: "Configuration"
+  role: "Role"
+  user: "User"
+  all: "All"
+  create: "Create"
+  group: "Group"
+  enabled: "Enabled"
+  installed: "Installed"
+  set: "Set"
+  table:
+    title:
+      currentLocation: "Current Location"
+      clone: "Clone"
+      promotableClone: "Promotable Clone"
+      masterSlave: "Master/Slave"
+      resourceLocationPreferences: "Resource Location Preferences"
+      nodeRule: "Node/Rule"
+      score: "Score"
+      resourceOrderingPreferences: "Resource Ordering Preferences"
+      resource: "Resource"
+      action: "Action"
+      beforeAfter: "Before/After"
+      resourceOrderingSetPreferences: "Resource Ordering Set Preferences"
+      preferenceNameSetofresources: "Preference Name/Set of Resources"
+      resourceTicketPreferences: "Resource Ticket Preferences"
+      ticket: "Ticket"
+      resourceRole: "Resource Role"
+      lossPolicy: "Loss Policy"
+      resourceTicketSetPreferences: "Resource Ticket Set Preferences"
+      resourceColocationPreferences: "Resource Colocation Preferences"
+      togetherApart: "Together/Apart"
+      resourceColocationSetPreferences: "Resource Colocation Set Preferences"
+      metaAttribute: "Meta Attribute"
+      value: "Value"
+      utilizationAttribute: "Utilization Attribute"
+      resourceGroup: "Resource Group"
+      disabled: "Disabled"
+      resourceID: "Resource ID"
+      fenceInstanceName: "Fence Instance Name"
+      uptime: "Uptime"
+      clusterDaemons: "Cluster Daemons"
+      runningResources: "Running Resources"
+      nodeAttributes: "Node Attributes"
+      startTheNode: "Start The Node"
+      doNotValidateWatchdog: "Do not validate watchdog"
+      informationAboutClusters: "Information About Clusters"
+      moreNodes: "More nodes..."
+      specifyWatchDevicesForNodes: "Specify watchdog devices for nodes"
+      ignoreOfflineNodes: "ignore offline nodes"
+      serviceStatus: "Service Status"
+      changeOrderOfResources: "Change order of resources"
+      usersAssigned: "Users Assigned"
+      groupsAssigned: "Groups Assigned"
+      PermissionsForCluster: "Permissions for cluster"
```

Comment:
Let's make this consistent and start all keys with a lover-case. There are a few other cases like this